src/api.v2/model/CellLevel.js

```diff
@@ -0,0 +1,40 @@
+const BasicModel = require('./BasicModel');
+const sqlClient = require('../../sql/sqlClient');
+const tableNames = require('./tableNames');
+const getLogger = require('../../utils/getLogger');
+
+const logger = getLogger();
+const fields = [
+  'id',
+  'name',
+  'upload_status',
+  'created_at',
+];
+
+class CellLevel extends BasicModel {
+  constructor(sql = sqlClient.get()) {
+    super(sql, tableNames.CELL_LEVEL, fields);
+  }
+
+  async getMetadataByExperimentId(experimentId) {
+    try {
+      const result = await this.sql
```

Comment:
You can do `return await`

Also, the `try` `catch` doesn't really change much I think? The error is shown when it throws, right?

Reply:
didnt see any problems without having the try/catch for now so i'll remove it.